accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -177,8 +175,6 @@ Windows-specific:
 
 * **NamedPipeClientStream**
     - `get_NumberOfServerInstances()`
-* **NamedPipeServerStream**
-    - `WaitForConnection()`
```

Comment:
it's supported on Unix: https://github.com/dotnet/runtime/blob/08643df14054d1273002254c909fa099649ebfcf/src/libraries/System.IO.Pipes/src/System/IO/Pipes/NamedPipeServerStream.Unix.cs#L54-L66

Reply:
Makes sense